### PR TITLE
Bug/epgr 8

### PR DESCRIPTION
**Pull request by Mihika0409**
Removed the empty link that was being shown if there was no equalTo concept.

### CODE REVIEW

File: grazer/src/main/webapp/WEB-INF/views/person.jsp [hunk 1]
```diff
@@ -32,9 +32,11 @@
 
 <div>
 	<c:forEach var="listVar" items="${concept.getEqualTo() }"> 
-		<c:if test="${listVar != '[]'}">
-	    		<a href="${listVar}" target="_blank"> <i class="fa fa-external-link" aria-hidden="true"></i>       ${listVar} </a><br/>
-	    	</c:if>
+		<c:if test="${not empty listVar }">
+			<c:if test="${listVar != '[]'}">
```
Comment (jdamerow) [inline grazer/src/main/webapp/WEB-INF/views/person.jsp]: let's remove this test. The equalTo field does sometimes contain [] but it's a mistake and shouldn't  be there. This way we can see it and fix it in conceptpower.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Review by jdamerow — APPROVED** [review]

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
retest this please

**Comment by jdamerow** [comment]
retest this please

**Review by abhikrish503501 — APPROVED** [review]

**Pull request merged**